### DISCUSSION TITLE
[@types/react-redux] .withTypes method is not exported

### DISCUSSION THREAD

**Discussion by jonueki** [DefinitelyTyped/DefinitelyTyped#69940] [Issues with a @types package]
Package Versions

    "react": "18.2.0",
    "react-dom": "18.2.0",
    "react-native": "0.72.12",
    "@reduxjs/toolkit": "^2.2.5",
    "react-redux": "^9.1.2",
    "@types/react-redux": "^7.1.33",

I want to use the function `useSelector.withTypes<>()` which is in the Redux [docs](https://redux.js.org/usage/usage-with-typescript#withtypes) but it doesn't seem to be exported in the [index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-redux/index.d.ts) file

I get the error 
```
Property 'withTypes' does not exist on type '<TState = DefaultRootState, TSelected = unknown>(selector: (state: TState) => TSelected, equalityFn?: ((left: TSelected, right: TSelected) => boolean) | undefined) => TSelected'.ts(2339)

```

[a comment by an automated account is not shown]

**Comment by markerikson** [comment]
React-Redux was rewritten in TypeScript as of 8.0, and we shipped the `withTypes()` method in 9.1.  You do not need `@types/react-redux` any more, and that is in fact the problem :) Please remove that from your dependencies.
